### PR TITLE
12.0 [MIG] web export view

### PR DESCRIPTION
**Pull request by valtterilattu**

### CODE REVIEW

File: web_export_view/static/src/xml/web_export_view_template.xml [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-<t t-name="WebExportTreeViewXls">
-    <div class="o_cp_sidebar btn-group btn-group-sm">
-        <button class="btn btn-default export_treeview_xls" type="button" title="Export xls">
-            <i t-translation="off" class="fa fa-file-excel-o"></i>
-        </button>
-    </div>
-</t>
+	<t t-name="WebExportTreeViewXls">
+    	<div class="o_cp_sidebar btn-group btn-group-sm">
+        	<button class="btn btn-default export_treeview_xls" type="button" title="Export xls">
```
Comment (astirpe) [inline web_export_view/static/src/xml/web_export_view_template.xml:5]: Please do not add tab characters, only spaces please.

**Review by astirpe — CHANGES_REQUESTED** [review]

**Review by Jortolsa-S73 — APPROVED** [review]
Functional review

**Comment by tarteo** [comment]
Have you considered to do this: https://github.com/OCA/web/issues/1048#issuecomment-432622633

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM, functional test.

File: web_export_view/static/src/xml/web_export_view_template.xml (hunk 1 above)
Comment (ovnicraft) [reply web_export_view/static/src/xml/web_export_view_template.xml:5]: @valtterilattu possible resolve this ? or @astirpe can you be permissive for tabs ?

**Comment by pedrobaeza** [comment]
Superseeded by #1269 as no response from last comments.

**Pull request closed without merging**